PCCard (PCMCIA/CardBus) support
Say Y here if you want to attach PCMCIA- or PC-cards to your Linux
computer.  These are credit-card size devices such as network cards,
modems or hard drives often used with laptops computers.  There are
actually two varieties of these cards: 16 bit PCMCIA and 32 bit
CardBus cards.

To compile this driver as modules, choose M here: the
module will be called pcmcia_core.
Depends: !UML
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:19:18.962610371 -0400
@@ -1999,9 +1999,6 @@
 CONFIG_BT_HCIBCM203X=m
 CONFIG_BT_HCIBPA10X=m
 CONFIG_BT_HCIBFUSB=m
-CONFIG_BT_HCIDTL1=m
-CONFIG_BT_HCIBT3C=m
-CONFIG_BT_HCIBLUECARD=m
 CONFIG_BT_HCIVHCI=m
 CONFIG_BT_MRVL=m
 CONFIG_BT_MRVL_SDIO=m
@@ -2213,23 +2210,7 @@
 CONFIG_CXL_MEM=m
 CONFIG_CXL_PORT=y
 CONFIG_CXL_SUSPEND=y
-CONFIG_PCCARD=y
-CONFIG_PCMCIA=y
-CONFIG_PCMCIA_LOAD_CIS=y
-CONFIG_CARDBUS=y
-
-#
-# PC-card bridges
-#
-CONFIG_YENTA=m
-CONFIG_YENTA_O2=y
-CONFIG_YENTA_RICOH=y
-CONFIG_YENTA_TI=y
-CONFIG_YENTA_ENE_TUNE=y
-CONFIG_YENTA_TOSHIBA=y
-CONFIG_PD6729=m
-CONFIG_I82092=m
-CONFIG_PCCARD_NONSTATIC=y
+# CONFIG_PCCARD is not set
 # CONFIG_RAPIDIO is not set
 
 #
@@ -2490,7 +2471,6 @@
 CONFIG_PARPORT_SERIAL=m
 # CONFIG_PARPORT_PC_FIFO is not set
 # CONFIG_PARPORT_PC_SUPERIO is not set
-CONFIG_PARPORT_PC_PCMCIA=m
 # CONFIG_PARPORT_AX88796 is not set
 CONFIG_PARPORT_1284=y
 CONFIG_PARPORT_NOT_PC=y
@@ -2774,7 +2754,6 @@
 CONFIG_SCSI_BFA_FC=m
 CONFIG_SCSI_VIRTIO=m
 CONFIG_SCSI_CHELSIO_FCOE=m
-# CONFIG_SCSI_LOWLEVEL_PCMCIA is not set
 CONFIG_SCSI_DH=y
 CONFIG_SCSI_DH_RDAC=m
 CONFIG_SCSI_DH_HP_SW=m
@@ -2870,7 +2849,6 @@
 CONFIG_PATA_MPIIX=m
 CONFIG_PATA_NS87410=m
 CONFIG_PATA_OPTI=m
-CONFIG_PATA_PCMCIA=m
 # CONFIG_PATA_RZ1000 is not set
 
 #
@@ -3069,8 +3047,6 @@
 CONFIG_ETHERNET=y
 CONFIG_MDIO=m
 CONFIG_NET_VENDOR_3COM=y
-CONFIG_PCMCIA_3C574=m
-CONFIG_PCMCIA_3C589=m
 CONFIG_VORTEX=m
 CONFIG_TYPHOON=m
 CONFIG_NET_VENDOR_ADAPTEC=y
@@ -3087,7 +3063,6 @@
 CONFIG_NET_VENDOR_AMD=y
 CONFIG_AMD8111_ETH=m
 CONFIG_PCNET32=m
-CONFIG_PCMCIA_NMCLAN=m
 CONFIG_AMD_XGBE=m
 CONFIG_AMD_XGBE_DCB=y
 CONFIG_AMD_XGBE_HAVE_ECC=y
@@ -3156,7 +3131,6 @@
 CONFIG_WINBOND_840=m
 CONFIG_DM9102=m
 CONFIG_ULI526X=m
-CONFIG_PCMCIA_XIRCOM=m
 CONFIG_NET_VENDOR_DLINK=y
 CONFIG_DL2K=m
 CONFIG_SUNDANCE=m
@@ -3172,7 +3146,6 @@
 CONFIG_TSNEP=m
 # CONFIG_TSNEP_SELFTESTS is not set
 # CONFIG_NET_VENDOR_EZCHIP is not set
-# CONFIG_NET_VENDOR_FUJITSU is not set
 CONFIG_NET_VENDOR_FUNGIBLE=y
 CONFIG_FUN_CORE=m
 CONFIG_FUN_ETH=m
@@ -3278,9 +3251,7 @@
 # CONFIG_NFP_APP_ABM_NIC is not set
 # CONFIG_NFP_DEBUG is not set
 CONFIG_NET_VENDOR_8390=y
-CONFIG_PCMCIA_AXNET=m
 CONFIG_NE2K_PCI=m
-CONFIG_PCMCIA_PCNET=m
 CONFIG_NET_VENDOR_NVIDIA=y
 CONFIG_FORCEDETH=m
 CONFIG_NET_VENDOR_OKI=y
@@ -3343,7 +3314,6 @@
 CONFIG_SFC_SIENA_SRIOV=y
 CONFIG_SFC_SIENA_MCDI_LOGGING=y
 CONFIG_NET_VENDOR_SMSC=y
-CONFIG_PCMCIA_SMC91C92=m
 CONFIG_EPIC100=m
 CONFIG_SMSC911X=m
 CONFIG_SMSC9420=m
@@ -3383,8 +3353,6 @@
 CONFIG_XILINX_EMACLITE=m
 # CONFIG_XILINX_AXI_EMAC is not set
 CONFIG_XILINX_LL_TEMAC=m
-CONFIG_NET_VENDOR_XIRCOM=y
-CONFIG_PCMCIA_XIRC2PS=m
 # CONFIG_FDDI is not set
 # CONFIG_HIPPI is not set
 # CONFIG_NET_SB1000 is not set
@@ -3680,8 +3648,6 @@
 # CONFIG_TMD_HERMES is not set
 CONFIG_NORTEL_HERMES=m
 CONFIG_PCI_HERMES=m
-CONFIG_PCMCIA_HERMES=m
-# CONFIG_PCMCIA_SPECTRUM is not set
 CONFIG_ORINOCO_USB=m
 CONFIG_P54_COMMON=m
 CONFIG_P54_USB=m
@@ -3691,7 +3657,6 @@
 CONFIG_WLAN_VENDOR_MARVELL=y
 CONFIG_LIBERTAS=m
 CONFIG_LIBERTAS_USB=m
-CONFIG_LIBERTAS_CS=m
 CONFIG_LIBERTAS_SDIO=m
 # CONFIG_LIBERTAS_SPI is not set
 # CONFIG_LIBERTAS_DEBUG is not set
@@ -3834,8 +3799,6 @@
 CONFIG_WLAN_VENDOR_QUANTENNA=y
 CONFIG_QTNFMAC=m
 CONFIG_QTNFMAC_PCIE=m
-# CONFIG_PCMCIA_RAYCS is not set
-# CONFIG_PCMCIA_WL3501 is not set
 CONFIG_MAC80211_HWSIM=m
 CONFIG_USB_NET_RNDIS_WLAN=m
 CONFIG_VIRT_WIFI=m
@@ -4196,7 +4159,6 @@
 CONFIG_SERIAL_8250_DMA=y
 CONFIG_SERIAL_8250_PCI=y
 CONFIG_SERIAL_8250_EXAR=m
-CONFIG_SERIAL_8250_CS=m
 CONFIG_SERIAL_8250_NR_UARTS=32
 CONFIG_SERIAL_8250_RUNTIME_UARTS=32
 CONFIG_SERIAL_8250_EXTENDED=y
@@ -4275,17 +4237,6 @@
 CONFIG_HW_RANDOM_VIRTIO=y
 CONFIG_HW_RANDOM_XIPHERA=m
 # CONFIG_APPLICOM is not set
-
-#
-# PCMCIA character devices
-#
-# CONFIG_SYNCLINK_CS is not set
-CONFIG_CARDMAN_4000=m
-CONFIG_CARDMAN_4040=m
-# CONFIG_SCR24X is not set
-CONFIG_IPWIRELESS=m
-# end of PCMCIA character devices
-
 CONFIG_MWAVE=m
 CONFIG_DEVMEM=y
 CONFIG_NVRAM=y
@@ -5082,8 +5033,6 @@
 CONFIG_SSB_PCIHOST_POSSIBLE=y
 CONFIG_SSB_PCIHOST=y
 CONFIG_SSB_B43_PCI_BRIDGE=y
-CONFIG_SSB_PCMCIAHOST_POSSIBLE=y
-CONFIG_SSB_PCMCIAHOST=y
 CONFIG_SSB_SDIOHOST_POSSIBLE=y
 CONFIG_SSB_SDIOHOST=y
 CONFIG_SSB_DRIVER_PCICORE_POSSIBLE=y
@@ -6580,7 +6529,6 @@
 CONFIG_SND_FIREWIRE_TASCAM=m
 CONFIG_SND_FIREWIRE_MOTU=m
 CONFIG_SND_FIREFACE=m
-# CONFIG_SND_PCMCIA is not set
 CONFIG_SND_SOC=m
 CONFIG_SND_SOC_AC97_BUS=y
 CONFIG_SND_SOC_GENERIC_DMAENGINE_PCM=y
@@ -7221,7 +7169,6 @@
 # CONFIG_USB_U132_HCD is not set
 CONFIG_USB_SL811_HCD=m
 CONFIG_USB_SL811_HCD_ISO=y
-# CONFIG_USB_SL811_CS is not set
 # CONFIG_USB_R8A66597_HCD is not set
 # CONFIG_USB_HCD_BCMA is not set
 # CONFIG_USB_HCD_SSB is not set
@@ -7444,7 +7391,6 @@
 CONFIG_MMC_ALCOR=m
 CONFIG_MMC_TIFM_SD=m
 # CONFIG_MMC_SPI is not set
-CONFIG_MMC_SDRICOH_CS=m
 CONFIG_MMC_CB710=m
 CONFIG_MMC_VIA_SDMMC=m
 CONFIG_MMC_VUB300=m
